README mwmichelpach1.md

```diff
@@ -1,5 +1,12 @@
 # Cloudera Director Public Scripts
 
+## Contributing Code
+
+In order to contribute code you must sign one of two agreements. If you are choosing to contribute on your own behalf then sign the ICLA agreement (https://github.com/cloudera/livy/wiki/Individual-Contributor-License-Agreement-(ICLA). If you are contributing on behalf of an organization or a company you must sign the CCLA agreement (https://github.com/cloudera/livy/wiki/Corporate-Contributor-License-Agreement-(CCLA). 
```

Comment:
Reusing the CCLA from Livy is no good, because it says it covers code contributed to Livy, and not to projects associated with Cloudera Director. See Schedule B in the CCLA and its use in the main body of the agreement.

We should either a) copy both agreements into a repo of our own (this one is fine) and edit the CCLA appropriately, or b) use generalized copies already available for any contributions to Cloudera projects, if there are any floating around for that purpose.